plugins/DevicesDetection/Reports/GetType.php

```diff
@@ -26,6 +27,8 @@ protected function init()
 
     public function configureView(ViewDataTable $view)
     {
+        $unknownTypeCount = 1;
+        $view->requestConfig->filter_limit = $unknownTypeCount + count(DeviceParser::getAvailableDeviceTypeNames());
```

Comment:
fyi @sgiehl: the Device detector `DeviceParser` is now API as far as Piwik is concerned